<!DOCTYPE html>
<html lang="en">
    <head>

        <!-- Google tag (gtag.js) -->
        <script async src="https://www.googletagmanager.com/gtag/js?id=G-WJRL86VLXW"></script>
        <script>
          window.dataLayer = window.dataLayer || [];
          function gtag(){dataLayer.push(arguments);}
          gtag('js', new Date());
          gtag('config', 'G-WJRL86VLXW');
        </script>

        <meta charset="utf-8">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <title>Publications - ReproNim</title>

        <!-- CSS -->
        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
        <link rel="stylesheet" href="https://code.ionicframework.com/ionicons/1.5.2/css/ionicons.min.css">
        <link rel="stylesheet" href="css/style.css">

        <!-- font -->
        <link href='https://fonts.googleapis.com/css?family=Nova+Square' rel='stylesheet'>
        <link rel="stylesheet" href="font/font.css">
    </head>
    <body>
        <div id="wrapper">
            <section class="section-1">
                <header class="site-header">
                    <div class="container">
                        <div class="row">
                            <div class="col-sm-4 col-xs-8">
                                <h1 class="logo"><a href="index.html">ReproNim</a></h1>
                            </div>
                            <div class="col-sm-8 col-xs-4">
                                <nav class="navbar pull-right" role="navigation">
                                    <!-- Brand and toggle get grouped for better mobile display -->
                                    <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
                                        <span class="ion-navicon"></span>
                                    </button>

                                    <!-- Collect the nav links, forms, and other content for toggling -->
                                    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
                                        <ul class="nav navbar-nav">
                                            <li><a href="discover.html">Discover</a></li>
                                            <li><a href="describe.html">Describe</a></li>
                                            <li><a href="do.html">Do</a></li>
                                            <li><a href="teach.html">Teach</a></li>
                                            <li><a href="fellowship.html">Fellowship</a></li>
                                            <li><a href="community.html">Community</a></li>
                                            <li><a href="search.html">üîç</a></li>
                                        </ul>
                                    </div><!-- /.navbar-collapse -->
                                </nav>
                            </div>
                        </div>  <!-- row -->
                    </div>
                </header>   <!-- site header -->
            </section>

            <section>
            	<div class="container">
                    <a href="https://www.repronim.org"><img src="images/logo-square-128.png" alt="ReproNim logo"></a>     
            		<div class="row">
            			<div class="col-md-12">
                            <h1 class="promo-text">ReproNim Publications</h1>
            				
            			</div>
            		</div><!-- row -->
            		<div class="row">
            			<div class="col-md-12">

                    
                    

          </div>
        </div>
      </div>
    </section>

    <section class="section-3">
        <div class="container">
            <div class="row">
                <div class="col-sm-12">
<!--
                    <h3>UPDATE section header</h3>
-->

<ul>
<li>Low, D. M., Rao, V., Randolph, G., Song, P. C., & Ghosh, S. S. (2024). Identifying bias in models that detect vocal fold paralysis from audio recordings using explainable machine learning and clinician ratings.  doi: <a href="https://doi.org/10.1101/2020.11.23.20235945">10.1101/2020.11.23.20235945</a>.</li>
<li>Burdinski, D., Kodibagkar, A., Potter, K., Schuster, R., Evins, A. E., Ghosh, S., & Gilman, J. (2024). Impact of year-long cannabis use for medical symptoms on brain activation during cognitive processes.  doi: <a href="https://doi.org/10.1101/2024.04.29.24306516">10.1101/2024.04.29.24306516</a>.</li>
<li>Lin, D. J., Backus, D., Chakraborty, S., Liew, S.-L., Valero-Cuevas, F. J., Patten, C., & Cotton, R. J. (2024). Transforming modeling in neurorehabilitation: clinical insights for personalized rehabilitation. Journal of NeuroEngineering and Rehabilitation, 21(1).  doi: <a href="https://doi.org/10.1186/s12984-024-01309-w">10.1186/s12984-024-01309-w</a>.</li>
<li>Renton, A. I., Dao, T. T., Johnstone, T., Civier, O., Sullivan, R. P., White, D. J., Lyons, P., Slade, B. M., Abbott, D. F., Amos, T. J., Bollmann, S., Botting, A., Campbell, M. E. J., Chang, J., Close, T. G., D√∂rig, M., Eckstein, K., Egan, G. F., Evas, S., ‚Ä¶ Bollmann, S. (2024). Neurodesk: an accessible, flexible and portable data analysis environment for reproducible neuroimaging. Nature Methods, 21(5), 804‚Äì808.  doi: <a href="https://doi.org/10.1038/s41592-023-02145-x">10.1038/s41592-023-02145-x</a>.</li>
<li>Szczepanik, M., Wagner, A. S., Heunis, S., Waite, L. K., Eickhoff, S. B., & Hanke, M. (2024). Teaching Research Data Management with DataLad: A Multi-year, Multi-domain Effort. Neuroinformatics, 22(4), 635‚Äì645.  doi: <a href="https://doi.org/10.1007/s12021-024-09665-7">10.1007/s12021-024-09665-7</a>.</li>
<li>Soko≈Çowski, A., Bhagwat, N., Chatelain, Y., Dugr√©, M., Hanganu, A., Monchi, O., McPherson, B., Wang, M., Poline, J.-B., Sharp, M., & Glatard, T. (2024). Longitudinal brain structure changes in Parkinson‚Äôs disease: A replication study. PLOS ONE, 19(1), e0295069.  doi: <a href="https://doi.org/10.1371/journal.pone.0295069">10.1371/journal.pone.0295069</a>.</li>
<li>Torabi, M., Mitsis, G. D., & Poline, J.-B. (2024). On the variability of dynamic functional connectivity assessment methods. GigaScience, 13.  doi: <a href="https://doi.org/10.1093/gigascience/giae009">10.1093/gigascience/giae009</a>.</li>
<li>Poldrack, R. A., Markiewicz, C. J., Appelhoff, S., Ashar, Y. K., Auer, T., Baillet, S., Bansal, S., Beltrachini, L., Benar, C. G., Bertazzoli, G., Bhogawar, S., Blair, R. W., Bortoletto, M., Boudreau, M., Brooks, T. L., Calhoun, V. D., Castelli, F. M., Clement, P., Cohen, A. L., ‚Ä¶ Gorgolewski, K. J. (2024). The past, present, and future of the brain imaging data structure (BIDS). Imaging Neuroscience, 2, 1‚Äì19.  doi: <a href="https://doi.org/10.1162/imag_a_00103">10.1162/imag_a_00103</a>.</li>
<li>Low, D. M., Rao, V., Randolph, G., Song, P. C., & Ghosh, S. S. (2024). Identifying bias in models that detect vocal fold paralysis from audio recordings using explainable machine learning and clinician ratings. PLOS Digital Health, 3(5), e0000516.  doi: <a href="https://doi.org/10.1371/journal.pdig.0000516">10.1371/journal.pdig.0000516</a>.</li>
<li>Halchenko, Y. O., Goncalves, M., Ghosh, S., Velasco, P., Visconti di Oleggio Castello, M., Salo, T., Wodder, J. T., Hanke, M., Sadil, P., Gorgolewski, K. J., Ioanas, H.-I., Rorden, C., Hendrickson, T. J., Dayan, M., Houlihan, S. D., Kent, J., Strauss, T., Lee, J., To, I., ‚Ä¶ Kennedy, D. N. (2024). HeuDiConv ‚Äî flexible DICOM conversion into structured directory layouts. Journal of Open Source Software, 9(99), 5839.  doi: <a href="https://doi.org/10.21105/joss.05839">10.21105/joss.05839</a>.</li>
<li>Hubbard, N. A., Bauer, C. C. C., Siless, V., Auerbach, R. P., Elam, J. S., Frosch, I. R., Henin, A., Hofmann, S. G., Hodge, M. R., Jones, R., Lenzini, P., Lo, N., Park, A. T., Pizzagalli, D. A., Vaz-DeSouza, F., Gabrieli, J. D. E., Whitfield-Gabrieli, S., Yendiki, A., & Ghosh, S. S. (2024). The Human Connectome Project of adolescent anxiety and depression dataset. Scientific Data, 11(1).  doi: <a href="https://doi.org/10.1038/s41597-024-03629-x">10.1038/s41597-024-03629-x</a>.</li>
<li>Ioanas, H.-I., Macdonald, A., & Halchenko, Y. O. (2024). Neuroimaging article reexecution and reproduction assessment system. Frontiers in Neuroinformatics, 18.  doi: <a href="https://doi.org/10.3389/fninf.2024.1376022">10.3389/fninf.2024.1376022</a>.</li>
<li>Queder, N., Tien, V. B., Abraham, S. A., Urchs, S. G. W., Helmer, K. G., Chaplin, D., van Erp, T. G. M., Kennedy, D. N., Poline, J.-B., Grethe, J. S., Ghosh, S. S., & Keator, D. B. (2023). NIDM-Terms: community-based terminology management for improved neuroimaging dataset descriptions and query. Frontiers in Neuroinformatics, 17.  doi: <a href="https://doi.org/10.3389/fninf.2023.1174156">10.3389/fninf.2023.1174156</a>.</li>
<li>Bollmann, S., Renton, A., Dao, T., Johnstone, T., Civier, O., Sullivan, R., White, D., Lyons, P., Slade, B., Abbott, D., Amos, T., Bollmann, S., Botting, A., Campbell, M., Chang, J., Close, T., Eckstein, K., Egan, G., Evas, S., ‚Ä¶ Narayanan, A. (2023). Neurodesk: An accessible, flexible, and portable data analysis environment for reproducible neuroimaging.  doi: <a href="https://doi.org/10.21203/rs.3.rs-2649734/v1">10.21203/rs.3.rs-2649734/v1</a>.</li>
<li>Larivi√®re, S., Bayrak, ≈û., Vos de Wael, R., Benkarim, O., Herholz, P., Rodriguez-Cruces, R., Paquola, C., Hong, S.-J., Misic, B., Evans, A. C., Valk, S. L., & Bernhardt, B. C. (2023). BrainStat: A toolbox for brain-wide statistics and multimodal feature associations. NeuroImage, 266, 119807.  doi: <a href="https://doi.org/10.1016/j.neuroimage.2022.119807">10.1016/j.neuroimage.2022.119807</a>.</li>
<li>Poline, J.-B., Das, S., Glatard, T., Madjar, C., Dickie, E. W., Lecours, X., Beaudry, T., Beck, N., Behan, B., Brown, S. T., Bujold, D., Beauvais, M., Caron, B., Czech, C., Dharsee, M., Dugr√©, M., Evans, K., Gee, T., Ippoliti, G., ‚Ä¶ Evans, A. C. (2023). Data and Tools Integration in the Canadian Open Neuroscience Platform. Scientific Data, 10(1).  doi: <a href="https://doi.org/10.1038/s41597-023-01946-1">10.1038/s41597-023-01946-1</a>.</li>
<li>Kiar, G., Clucas, J., Feczko, E., Goncalves, M., Jarecka, D., Markiewicz, C. J., Halchenko, Y. O., Hermosillo, R., Li, X., Miranda-Dominguez, O., Ghosh, S., Poldrack, R. A., Satterthwaite, T. D., Milham, M. P., & Fair, D. (2023). Align with the NMIND consortium for better neuroimaging. Nature Human Behaviour, 7(7), 1027‚Äì1028.  doi: <a href="https://doi.org/10.1038/s41562-023-01647-0">10.1038/s41562-023-01647-0</a>.</li>
<li>Torabian, S., V√©lez, N., Sochat, V., Halchenko, Y. O., & Grossman, E. D. (2023). The PyMVPA BIDS-App: a robust multivariate pattern analysis pipeline for fMRI data. Frontiers in Neuroscience, 17.  doi: <a href="https://doi.org/10.3389/fnins.2023.1233416">10.3389/fnins.2023.1233416</a>.</li>
<li>Wang, Q., Aljassar, M., Bhagwat, N., Zeighami, Y., Evans, A. C., Dagher, A., Pike, G. B., Sadikot, A. F., & Poline, J.-B. (2023). Reproducibility of cerebellar involvement as quantified by consensus structural MRI biomarkers in advanced essential tremor. Scientific Reports, 13(1).  doi: <a href="https://doi.org/10.1038/s41598-022-25306-y">10.1038/s41598-022-25306-y</a>.</li>
<li>Peraza, J. A., Salo, T., Riedel, M. C., Bottenhorn, K. L., Poline, J.-B., Dock√®s, J., Kent, J. D., Bartley, J. E., Flannery, J. S., Hill-Bowen, L. D., Lobo, R. P., Poudel, R., Ray, K. L., Robinson, J. L., Laird, R. W., Sutherland, M. T., de la Vega, A., & Laird, A. R. (2023). Methods for decoding cortical gradients of functional connectivity.  doi: <a href="https://doi.org/10.1101/2023.08.01.551505">10.1101/2023.08.01.551505</a>.</li>
<li>Zhao, C., Jarecka, D., Covitz, S., Chen, Y., Eickhoff, S. B., Fair, D. A., Franco, A. R., Halchenko, Y. O., Hendrickson, T. J., Hoffstaedter, F., Houghton, A., Kiar, G., Macdonald, A., Mehta, K., Milham, M. P., Salo, T., Hanke, M., Ghosh, S. S., Cieslak, M., & Satterthwaite, T. D. (2023). A reproducible and generalizable software workflow for analysis of large-scale neuroimaging data collections using BIDS Apps.  doi: <a href="https://doi.org/10.1101/2023.08.16.552472">10.1101/2023.08.16.552472</a>.</li>
<li>Notter, M. P., Herholz, P., Da Costa, S., Gulban, O. F., Isik, A. I., Gaglianese, A., & Murray, M. M. (2022). fMRIflows: A Consortium of Fully Automatic Univariate and Multivariate fMRI Processing Pipelines. Brain Topography, 36(2), 172‚Äì191.  doi: <a href="https://doi.org/10.1007/s10548-022-00935-8">10.1007/s10548-022-00935-8</a>.</li>
<li>Ferris, J. K., Lo, B. P., Khlif, M. S., Brodtmann, A., Boyd, L. A., & Liew, S.-L. (2023). Optimizing automated white matter hyperintensity segmentation in individuals with stroke. Frontiers in Neuroimaging, 2.  doi: <a href="https://doi.org/10.3389/fnimg.2023.1099301">10.3389/fnimg.2023.1099301</a>.</li>
<li>Makris, N., Rushmore, R., Kaiser, J., Albaugh, M., Kubicki, M., Rathi, Y., Zhang, F., O‚ÄôDonnell, L. J., Yeterian, E., Caviness, V. S., & Kennedy, D. N. (2023). A Proposed Human Structural Brain Connectivity Matrix in the Center for Morphometric Analysis Harvard-Oxford Atlas Framework: A Historical Perspective and Future Direction for Enhancing the Precision of Human Structural Connectivity with a Novel Neuroanatomical Typology. Developmental Neuroscience, 45(4), 161‚Äì180. Portico.  doi: <a href="https://doi.org/10.1159/000530358">10.1159/000530358</a>.</li>
<li>Modarres, M., Cochran, D., Kennedy, D. N., & Frazier, J. A. (2023). Comparison of comprehensive quantitative EEG metrics between typically developing boys and girls in resting state eyes-open and eyes-closed conditions. Frontiers in Human Neuroscience, 17.  doi: <a href="https://doi.org/10.3389/fnhum.2023.1237651">10.3389/fnhum.2023.1237651</a>.</li>
<li>Poline, J.-B., Kennedy, D. N., Sommer, F. T., Ascoli, G. A., Van Essen, D. C., Ferguson, A. R., Grethe, J. S., Hawrylycz, M. J., Thompson, P. M., Poldrack, R. A., Ghosh, S. S., Keator, D. B., Athey, T. L., Vogelstein, J. T., Mayberg, H. S., & Martone, M. E. (2022). Is Neuroscience FAIR? A Call for Collaborative Standardisation of Neuroscience Data. Neuroinformatics, 20(2), 507‚Äì512.  doi: <a href="https://doi.org/10.1007/s12021-021-09557-0">10.1007/s12021-021-09557-0</a>.</li>
<li>Royer, J., Rodr√≠guez-Cruces, R., Tavakol, S., Larivi√®re, S., Herholz, P., Li, Q., Vos de Wael, R., Paquola, C., Benkarim, O., Park, B., Lowe, A. J., Margulies, D., Smallwood, J., Bernasconi, A., Bernasconi, N., Frauscher, B., & Bernhardt, B. C. (2022). An Open MRI Dataset For Multiscale Neuroscience. Scientific Data, 9(1).  doi: <a href="https://doi.org/10.1038/s41597-022-01682-y">10.1038/s41597-022-01682-y</a>.</li>
<li>Torres-Esp√≠n, A., Almeida, C. A., Chou, A., Huie, J. R., Chiu, M., Vavrek, R., Sacramento, J., Orr, M. B., Gensel, J. C., Grethe, J. S., Martone, M. E., Fouad, K., Ferguson, A. R., Alilain, W., Bacon, M., Batty, N., Beattie, M., Bresnahan, J., ‚Ä¶ Zholudeva, L. (2021). Promoting FAIR Data Through Community-driven Agile Design: the Open Data Commons for Spinal Cord Injury (odc-sci.org). Neuroinformatics, 20(1), 203‚Äì219.  doi: <a href="https://doi.org/10.1007/s12021-021-09533-8">10.1007/s12021-021-09533-8</a>.</li>
<li>Abrams, M. B., Bjaalie, J. G., Das, S., Egan, G. F., Ghosh, S. S., Goscinski, W. J., Grethe, J. S., Kotaleski, J. H., Ho, E. T. W., Kennedy, D. N., Lanyon, L. J., Leergaard, T. B., Mayberg, H. S., Milanesi, L., Mouƒçek, R., Poline, J. B., Roy, P. K., Strother, S. C., Tang, T. B., ‚Ä¶ Martone, M. E. (2021). A Standards Organization for Open and FAIR Neuroscience: the International Neuroinformatics Coordinating Facility. Neuroinformatics, 20(1), 25‚Äì36.  doi: <a href="https://doi.org/10.1007/s12021-020-09509-0">10.1007/s12021-020-09509-0</a>.</li>
<li>Perez-Lebel, A., Varoquaux, G., Le¬†Morvan, M., Josse, J., & Poline, J.-B. (2022). Benchmarking missing-values approaches for predictive models on health databases. GigaScience, 11.  doi: <a href="https://doi.org/10.1093/gigascience/giac013">10.1093/gigascience/giac013</a>.</li>
<li>Kumar, A., Crowley, A., Queder, N., Poline, J., Ghosh, S. S., Kennedy, D., Grethe, J. S., Helmer, K. G., & Keator, D. B. (2022). The Neuroimaging Data Model Linear Regression Tool (nidm_linreg): PyNIDM Project. F1000Research, 11, 228.  doi: <a href="https://doi.org/10.12688/f1000research.108008.2">10.12688/f1000research.108008.2</a>.</li>
<li>DuPre, E., Holdgraf, C., Karakuzu, A., Tetrel, L., Bellec, P., Stikov, N., & Poline, J.-B. (2022). Beyond advertising: New infrastructures for publishing integrated research objects. PLOS Computational Biology, 18(1), e1009651.  doi: <a href="https://doi.org/10.1371/journal.pcbi.1009651">10.1371/journal.pcbi.1009651</a>.</li>
<li>Satz, S., Halchenko, Y. O., Ragozzino, R., Lucero, M. M., Phillips, M. L., Swartz, H. A., & Manelis, A. (2022). The Relationship Between Default Mode and Dorsal Attention Networks Is Associated With Depressive Disorder Diagnosis and the Strength of Memory Representations Acquired Prior to the Resting State Scan. Frontiers in Human Neuroscience, 16.  doi: <a href="https://doi.org/10.3389/fnhum.2022.749767">10.3389/fnhum.2022.749767</a>.</li>
<li>Cruces, R. R., Royer, J., Herholz, P., Larivi√®re, S., Vos de Wael, R., Paquola, C., Benkarim, O., Park, B., Degr√©-Pelletier, J., Nelson, M. C., DeKraker, J., Leppert, I. R., Tardif, C., Poline, J.-B., Concha, L., & Bernhardt, B. C. (2022). Micapipe: A pipeline for multimodal neuroimaging and connectome analysis. NeuroImage, 263, 119612.  doi: <a href="https://doi.org/10.1016/j.neuroimage.2022.119612">10.1016/j.neuroimage.2022.119612</a>.</li>
<li>Niso, G., Botvinik-Nezer, R., Appelhoff, S., De La Vega, A., Esteban, O., Etzel, J. A., Finc, K., Ganz, M., Gau, R., Halchenko, Y. O., Herholz, P., Karakuzu, A., Keator, D. B., Markiewicz, C. J., Maumet, C., Pernet, C. R., Pestilli, F., Queder, N., Schmitt, T., ‚Ä¶ Rieger, J. W. (2022). Open and reproducible neuroimaging: From study inception to publication. NeuroImage, 263, 119623.  doi: <a href="https://doi.org/10.1016/j.neuroimage.2022.119623">10.1016/j.neuroimage.2022.119623</a>.</li>
<li>Manelis, A., Halchenko, Y. O., Bonar, L., Stiffler, R. S., Satz, S., Miceli, R., Ladouceur, C. D., Bebko, G., Iyengar, S., Swartz, H. A., & Phillips, M. L. (2022). Working memory updating in individuals with bipolar and unipolar depression: fMRI study. Translational Psychiatry, 12(1).  doi: <a href="https://doi.org/10.1038/s41398-022-02211-6">10.1038/s41398-022-02211-6</a>.</li>
<li>Modarres, M., Cochran, D., Kennedy, D. N., Schmidt, R., Fitzpatrick, P., & Frazier, J. A. (2021). Biomarkers Based on Comprehensive Hierarchical EEG Coherence Analysis: Example Application to Social Competence in Autism (Preliminary Results). Neuroinformatics, 20(1), 53‚Äì62.  doi: <a href="https://doi.org/10.1007/s12021-021-09517-8">10.1007/s12021-021-09517-8</a>.</li>
<li>McNaughton, R., Pieper, C., Sakai, O., Rollins, J. V., Zhang, X., Kennedy, D. N., Frazier, J. A., Douglass, L., Heeren, T., Fry, R. C., O‚ÄôShea, T. M., Kuban, K. K., Jara, H., Rollins, J. V., Shah, B., Singh, R., Vaidya, R., Van Marter, L., ‚Ä¶ Vogt, K. (2022). Quantitative MRI Characterization of the Extremely Preterm Brain at                     Adolescence: Atypical versus Neurotypical Developmental Pathways. Radiology, 304(2), 419‚Äì428.  doi: <a href="https://doi.org/10.1148/radiol.210385">10.1148/radiol.210385</a>.</li>
<li>de la Vega, A., Rocca, R., Blair, R. W., Markiewicz, C. J., Mentch, J., Kent, J. D., Herholz, P., Ghosh, S. S., Poldrack, R. A., & Yarkoni, T. (2022). Neuroscout, a unified platform for generalizable and reproducible fMRI research. ELife, 11. CLOCKSS.  doi: <a href="https://doi.org/10.7554/elife.79277">10.7554/eLife.79277</a>.</li>
<li>Ciric, R., Thompson, W. H., Lorenz, R., Goncalves, M., MacNicol, E. E., Markiewicz, C. J., Halchenko, Y. O., Ghosh, S. S., Gorgolewski, K. J., Poldrack, R. A., & Esteban, O. (2022). TemplateFlow: FAIR-sharing of multi-scale, multi-species brain models. Nature Methods, 19(12), 1568‚Äì1571.  doi: <a href="https://doi.org/10.1038/s41592-022-01681-2">10.1038/s41592-022-01681-2</a>.</li>
<li>Eke, D. O., Bernard, A., Bjaalie, J. G., Chavarriaga, R., Hanakawa, T., Hannan, A. J., Hill, S. L., Martone, M. E., McMahon, A., Ruebel, O., Crook, S., Thiels, E., & Pestilli, F. (2022). International data governance for neuroscience. Neuron, 110(4), 600‚Äì612.  doi: <a href="https://doi.org/10.1016/j.neuron.2021.11.017">10.1016/j.neuron.2021.11.017</a>.</li>
<li>Boaro, A., Kaczmarzyk, J. R., Kavouridis, V. K., Harary, M., Mammi, M., Dawood, H., Shea, A., Cho, E. Y., Juvekar, P., Noh, T., Rana, A., Ghosh, S., & Arnaout, O. (2022). Deep neural networks allow expert-level brain meningioma segmentation and present potential for improvement of clinical practice. Scientific Reports, 12(1).  doi: <a href="https://doi.org/10.1038/s41598-022-19356-5">10.1038/s41598-022-19356-5</a>.</li>
<li>Lo, B. P., Donnelly, M. R., Barisano, G., & Liew, S.-L. (2023). A standardized protocol for manually segmenting stroke lesions on high-resolution T1-weighted MR images. Frontiers in Neuroimaging, 1.  doi: <a href="https://doi.org/10.3389/fnimg.2022.1098604">10.3389/fnimg.2022.1098604</a>.</li>
<li>Baranger, D. A. A., Halchenko, Y. O., Satz, S., Ragozzino, R., Iyengar, S., Swartz, H. A., & Manelis, A. (2021). Protocol for a machine learning algorithm predicting depressive disorders using the T1w/T2w ratio. MethodsX, 8, 101595.  doi: <a href="https://doi.org/10.1016/j.mex.2021.101595">10.1016/j.mex.2021.101595</a>.</li>
<li>Markiewicz, C. J., Gorgolewski, K. J., Feingold, F., Blair, R., Halchenko, Y. O., Miller, E., Hardcastle, N., Wexler, J., Esteban, O., Goncavles, M., Jwa, A., & Poldrack, R. (2021). The OpenNeuro resource for sharing of neuroscience data. ELife, 10. CLOCKSS.  doi: <a href="https://doi.org/10.7554/elife.71774">10.7554/eLife.71774</a>.</li>
<li>Bannier, E., Barker, G., Borghesani, V., Broeckx, N., Clement, P., Emblem, K. E., Ghosh, S., Glerean, E., Gorgolewski, K. J., Havu, M., Halchenko, Y. O., Herholz, P., Hespel, A., Heunis, S., Hu, Y., Hu, C., Huijser, D., de la Iglesia Vay√°, M., Jancalek, R., ‚Ä¶ Zhu, H. (2021). The Open Brain Consent: Informing research participants and obtaining consent to share brain imaging data. Human Brain Mapping, 42(7), 1945‚Äì1951. Portico.  doi: <a href="https://doi.org/10.1002/hbm.25351">10.1002/hbm.25351</a>.</li>
<li>Baranger, D. A. A., Halchenko, Y. O., Satz, S., Ragozzino, R., Iyengar, S., Swartz, H. A., & Manelis, A. (2021). Aberrant levels of cortical myelin distinguish individuals with depressive disorders from healthy controls. NeuroImage: Clinical, 32, 102790.  doi: <a href="https://doi.org/10.1016/j.nicl.2021.102790">10.1016/j.nicl.2021.102790</a>.</li>
<li>Halchenko, Y., Meyer, K., Poldrack, B., Solanky, D., Wagner, A., Gors, J., MacFarlane, D., Pustina, D., Sochat, V., Ghosh, S., M√∂nch, C., Markiewicz, C., Waite, L., Shlyakhter, I., de la Vega, A., Hayashi, S., H√§usler, C., Poline, J.-B., Kadelka, T., ‚Ä¶ Hanke, M. (2021). DataLad: distributed system for joint management of code, data, and their relationship. Journal of Open Source Software, 6(63), 3262.  doi: <a href="https://doi.org/10.21105/joss.03262">10.21105/joss.03262</a>.</li>
<li>Manelis, A., Soehner, A., Halchenko, Y. O., Satz, S., Ragozzino, R., Lucero, M., Swartz, H. A., Phillips, M. L., & Versace, A. (2021). White matter abnormalities in adults with bipolar disorder type-II and unipolar depression. Scientific Reports, 11(1).  doi: <a href="https://doi.org/10.1038/s41598-021-87069-2">10.1038/s41598-021-87069-2</a>.</li>
<li>McAvoy, M., Prieto, P. C., Kaczmarzyk, J. R., Fern√°ndez, I. S., McNulty, J., Smith, T., Yu, K.-H., Gormley, W. B., & Arnaout, O. (2021). Classification of glioblastoma versus primary central nervous system lymphoma using convolutional neural networks. Scientific Reports, 11(1).  doi: <a href="https://doi.org/10.1038/s41598-021-94733-0">10.1038/s41598-021-94733-0</a>.</li>
<li>Gau, R., Noble, S., Heuer, K., Bottenhorn, K. L., Bilgin, I. P., Yang, Y.-F., Huntenburg, J. M., Bayer, J. M. M., Bethlehem, R. A. I., Rhoads, S. A., Vogelbacher, C., Borghesani, V., Levitis, E., Wang, H.-T., Van Den Bossche, S., Kobeleva, X., Legarreta, J. H., Guay, S., Atay, S. M., ‚Ä¶ Zuo, X.-N. (2021). Brainhack: Developing a culture of open, inclusive, community-driven neuroscience. Neuron, 109(11), 1769‚Äì1775.  doi: <a href="https://doi.org/10.1016/j.neuron.2021.04.001">10.1016/j.neuron.2021.04.001</a>.</li>
<li>Abrams, M. B., Bjaalie, J. G., Das, S., Egan, G. F., Ghosh, S. S., Goscinski, W. J., Grethe, J. S., Kotaleski, J. H., Ho, E. T. W., Kennedy, D. N., Lanyon, L. J., Leergaard, T. B., Mayberg, H. S., Milanesi, L., Mouƒçek, R., Poline, J. B., Roy, P. K., Strother, S. C., Tang, T. B., ‚Ä¶ Martone, M. E. (2021). Correction to: A Standards Organization for Open and FAIR Neuroscience: the International Neuroinformatics Coordinating Facility. Neuroinformatics, 20(1), 37‚Äì38.  doi: <a href="https://doi.org/10.1007/s12021-021-09522-x">10.1007/s12021-021-09522-x</a>.</li>
<li>Hanke, M., Pestilli, F., Wagner, A. S., Markiewicz, C. J., Poline, J.-B., & Halchenko, Y. O. (2021). In defense of decentralized research data management. Neuroforum, 0(0).  doi: <a href="https://doi.org/10.1515/nf-2020-0037">10.1515/nf-2020-0037</a>.</li>
<li>Klein, A., Clucas, J., Krishnakumar, A., Ghosh, S. S., Van Auken, W., Thonet, B., Sabram, I., Acuna, N., Keshavan, A., Rossiter, H., Xiao, Y., Semenuta, S., Badioli, A., Konishcheva, K., Abraham, S. A., Alexander, L. M., Merikangas, K. R., Swendsen, J., Lindner, A. B., & Milham, M. P. (2021). Remote Digital Psychiatry for Mobile Mental Health Assessment and Therapy: MindLogger Platform Development Study. Journal of Medical Internet Research, 23(11), e22369.  doi: <a href="https://doi.org/10.2196/22369">10.2196/22369</a>.</li>
<li>Nastase, S. A., Liu, Y.-F., Hillman, H., Zadbood, A., Hasenfratz, L., Keshavarzian, N., Chen, J., Honey, C. J., Yeshurun, Y., Regev, M., Nguyen, M., Chang, C. H. C., Baldassano, C., Lositsky, O., Simony, E., Chow, M. A., Leong, Y. C., Brooks, P. P., Micciche, E., ‚Ä¶ Hasson, U. (2021). The ‚ÄúNarratives‚Äù fMRI dataset for evaluating models of naturalistic language comprehension. Scientific Data, 8(1).  doi: <a href="https://doi.org/10.1038/s41597-021-01033-3">10.1038/s41597-021-01033-3</a>.</li>
<li>Dock√®s, J., Varoquaux, G., & Poline, J.-B. (2021). Preventing dataset shift from breaking machine-learning biomarkers. GigaScience, 10(9).  doi: <a href="https://doi.org/10.1093/gigascience/giab055">10.1093/gigascience/giab055</a>.</li>
<li>Bhagwat, N., Barry, A., Dickie, E. W., Brown, S. T., Devenyi, G. A., Hatano, K., DuPre, E., Dagher, A., Chakravarty, M., Greenwood, C. M. T., Misic, B., Kennedy, D. N., & Poline, J.-B. (2021). Understanding the impact of preprocessing pipelines on neuroimaging cortical surface analyses. GigaScience, 10(1).  doi: <a href="https://doi.org/10.1093/gigascience/giaa155">10.1093/gigascience/giaa155</a>.</li>
<li>Bazeille, T., DuPre, E., Richard, H., Poline, J.-B., & Thirion, B. (2021). An empirical evaluation of functional alignment using inter-subject decoding. NeuroImage, 245, 118683.  doi: <a href="https://doi.org/10.1016/j.neuroimage.2021.118683">10.1016/j.neuroimage.2021.118683</a>.</li>
<li>Markello, R. D., Arnatkeviciute, A., Poline, J.-B., Fulcher, B. D., Fornito, A., & Misic, B. (2021). Standardizing workflows in imaging transcriptomics with the abagen toolbox. ELife, 10. CLOCKSS.  doi: <a href="https://doi.org/10.7554/elife.72129">10.7554/eLife.72129</a>.</li>
<li>Gan-Or, Z., Rao, T., Leveille, E., Degroot, C., Chouinard, S., Cicchetti, F., Dagher, A., Das, S., Desautels, A., Drouin-Ouellet, J., Durcan, T., Gagnon, J.-F., Genge, A., Karamchandani, J., Lafontaine, A.-L., Sun, S. L. W., Langlois, M., Levesque, M., Melmed, C., ‚Ä¶ Fon, E. A. (2020). The Quebec Parkinson Network: A Researcher-Patient Matching Platform and Multimodal Biorepository. Journal of Parkinson‚Äôs Disease, 10(1), 301‚Äì313.  doi: <a href="https://doi.org/10.3233/jpd-191775">10.3233/JPD-191775</a>.</li>
<li>Hung, Y., Uchida, M., Gaillard, S. L., Woodworth, H., Kelberman, C., Capella, J., Kadlec, K., Goncalves, M., Ghosh, S., Yendiki, A., Chai, X. J., Hirshfeld-Becker, D. R., Whitfield-Gabrieli, S., Gabrieli, J. D. E., & Biederman, J. (2020). Cingulum-Callosal white-matter microstructure associated with emotional dysregulation in children: A diffusion tensor imaging study. NeuroImage: Clinical, 27, 102266.  doi: <a href="https://doi.org/10.1016/j.nicl.2020.102266">10.1016/j.nicl.2020.102266</a>.</li>
<li>Cheng, C. P., & Halchenko, Y. O. (2020). A new virtue of phantom MRI data: explaining variance in human participant data. F1000Research, 9, 1131.  doi: <a href="https://doi.org/10.12688/f1000research.24544.1">10.12688/f1000research.24544.1</a>.</li>
<li>Hubbard, N. A., Siless, V., Frosch, I. R., Goncalves, M., Lo, N., Wang, J., Bauer, C. C. C., Conroy, K., Cosby, E., Hay, A., Jones, R., Pinaire, M., Vaz De Souza, F., Vergara, G., Ghosh, S., Henin, A., Hirshfeld-Becker, D. R., Hofmann, S. G., Rosso, I. M., ‚Ä¶ Whitfield-Gabrieli, S. (2020). Brain function and clinical characterization in the Boston adolescent neuroimaging of depression and anxiety study. NeuroImage: Clinical, 27, 102240.  doi: <a href="https://doi.org/10.1016/j.nicl.2020.102240">10.1016/j.nicl.2020.102240</a>.</li>
<li>Low, D. M., Bentley, K. H., & Ghosh, S. S. (2020). Automated assessment of psychiatric disorders using speech: A systematic review. Laryngoscope Investigative Otolaryngology, 5(1), 96‚Äì116. Portico.  doi: <a href="https://doi.org/10.1002/lio2.354">10.1002/lio2.354</a>.</li>
<li>Charles, A. S., Falk, B., Turner, N., Pereira, T. D., Tward, D., Pedigo, B. D., Chung, J., Burns, R., Ghosh, S. S., Kebschull, J. M., Silversmith, W., & Vogelstein, J. T. (2020). Toward Community-Driven Big Open Brain Science: Open Big Data and Tools for Structure, Function, and Genetics. Annual Review of Neuroscience, 43(1), 441‚Äì464.  doi: <a href="https://doi.org/10.1146/annurev-neuro-100119-110036">10.1146/annurev-neuro-100119-110036</a>.</li>
<li>Hodge, S. M., Haselgrove, C., Honor, L., Kennedy, D. N., & Frazier, J. A. (2021). An assessment of the autism neuroimaging literature for the prospects of re-executability. F1000Research, 9, 1031.  doi: <a href="https://doi.org/10.12688/f1000research.25306.2">10.12688/f1000research.25306.2</a>.</li>
<li>Siless, V., Hubbard, N. A., Jones, R., Wang, J., Lo, N., Bauer, C. C. C., Goncalves, M., Frosch, I., Norton, D., Vergara, G., Conroy, K., De Souza, F. V., Rosso, I. M., Wickham, A. H., Cosby, E. A., Pinaire, M., Hirshfeld-Becker, D., Pizzagalli, D. A., Henin, A., ‚Ä¶ Yendiki, A. (2020). Image acquisition and quality assurance in the Boston Adolescent Neuroimaging of Depression and Anxiety study. NeuroImage: Clinical, 26, 102242.  doi: <a href="https://doi.org/10.1016/j.nicl.2020.102242">10.1016/j.nicl.2020.102242</a>.</li>
<li>DuPre, E., Hanke, M., & Poline, J.-B. (2020). Nature abhors a paywall: How open science can realize the potential of naturalistic stimuli. NeuroImage, 216, 116330.  doi: <a href="https://doi.org/10.1016/j.neuroimage.2019.116330">10.1016/j.neuroimage.2019.116330</a>.</li>
<li>Esteban, O., Ciric, R., Finc, K., Blair, R. W., Markiewicz, C. J., Moodie, C. A., Kent, J. D., Goncalves, M., DuPre, E., Gomez, D. E. P., Ye, Z., Salo, T., Valabregue, R., Amlien, I. K., Liem, F., Jacoby, N., Stojiƒá, H., Cieslak, M., Urchs, S., ‚Ä¶ Gorgolewski, K. J. (2020). Analysis of task-based functional MRI data preprocessed with fMRIPrep. Nature Protocols, 15(7), 2186‚Äì2202.  doi: <a href="https://doi.org/10.1038/s41596-020-0327-3">10.1038/s41596-020-0327-3</a>.</li>
<li>Botvinik-Nezer, R., Holzmeister, F., Camerer, C. F., Dreber, A., Huber, J., Johannesson, M., Kirchler, M., Iwanir, R., Mumford, J. A., Adcock, R. A., Avesani, P., Baczkowski, B. M., Bajracharya, A., Bakst, L., Ball, S., Barilari, M., Bault, N., Beaton, D., Beitner, J., ‚Ä¶ Schonberg, T. (2020). Variability in the analysis of a single neuroimaging dataset by many teams. Nature, 582(7810), 84‚Äì88.  doi: <a href="https://doi.org/10.1038/s41586-020-2314-9">10.1038/s41586-020-2314-9</a>.</li>
<li>Lin, D., Crabtree, J., Dillo, I., Downs, R. R., Edmunds, R., Giaretta, D., De Giusti, M., L‚ÄôHours, H., Hugo, W., Jenkyns, R., Khodiyar, V., Martone, M. E., Mokrane, M., Navale, V., Petters, J., Sierman, B., Sokolova, D. V., Stockhause, M., & Westbrook, J. (2020). The TRUST Principles for digital repositories. Scientific Data, 7(1).  doi: <a href="https://doi.org/10.1038/s41597-020-0486-7">10.1038/s41597-020-0486-7</a>.</li>
<li>Keshavan, A., & Poline, J.-B. (2019). From the Wet Lab to the Web Lab: A Paradigm Shift in Brain Imaging Research. Frontiers in Neuroinformatics, 13.  doi: <a href="https://doi.org/10.3389/fninf.2019.00003">10.3389/fninf.2019.00003</a>.</li>
<li>Hagler, D. J., Hatton, SeanN., Cornejo, M. D., Makowski, C., Fair, D. A., Dick, A. S., Sutherland, M. T., Casey, B. J., Barch, D. M., Harms, M. P., Watts, R., Bjork, J. M., Garavan, H. P., Hilmer, L., Pung, C. J., Sicat, C. S., Kuperman, J., Bartsch, H., Xue, F., ‚Ä¶ Dale, A. M. (2019). Image processing and analysis methods for the Adolescent Brain Cognitive Development Study. NeuroImage, 202, 116091.  doi: <a href="https://doi.org/10.1016/j.neuroimage.2019.116091">10.1016/j.neuroimage.2019.116091</a>.</li>
<li>Kennedy, D. N., Abraham, S. A., Bates, J. F., Crowley, A., Ghosh, S., Gillespie, T., Goncalves, M., Grethe, J. S., Halchenko, Y. O., Hanke, M., Haselgrove, C., Hodge, S. M., Jarecka, D., Kaczmarzyk, J., Keator, D. B., Meyer, K., Martone, M. E., Padhy, S., Poline, J.-B., ‚Ä¶ Travers, M. (2019). Everything Matters: The ReproNim Perspective on Reproducible Neuroimaging. Frontiers in Neuroinformatics, 13.  doi: <a href="https://doi.org/10.3389/fninf.2019.00001">10.3389/fninf.2019.00001</a>.</li>
<li>Yarkoni, T., Markiewicz, C., de la Vega, A., Gorgolewski, K., Salo, T., Halchenko, Y., McNamara, Q., DeStasio, K., Poline, J.-B., Petrov, D., Hayot-Sasson, V., Nielson, D., Carlin, J., Kiar, G., Whitaker, K., DuPre, E., Wagner, A., Tirrell, L., Jas, M., ‚Ä¶ Blair, R. (2019). PyBIDS: Python tools for BIDS datasets. Journal of Open Source Software, 4(40), 1294.  doi: <a href="https://doi.org/10.21105/joss.01294">10.21105/joss.01294</a>.</li>
<li>Guell, X., Goncalves, M., Kaczmarzyk, J. R., Gabrieli, J. D. E., Schmahmann, J. D., & Ghosh, S. S. (2019). LittleBrain: A gradient-based tool for the topographical interpretation of cerebellar neuroimaging findings. PLOS ONE, 14(1), e0210028.  doi: <a href="https://doi.org/10.1371/journal.pone.0210028">10.1371/journal.pone.0210028</a>.</li>
<li>Poline, J.-B. (2019). From data sharing to data publishing. MNI Open Research, 2, 1.  doi: <a href="https://doi.org/10.12688/mniopenres.12772.2">10.12688/mniopenres.12772.2</a>.</li>
<li>Sitek, K. R., Gulban, O. F., Calabrese, E., Johnson, G. A., Lage-Castellanos, A., Moerel, M., Ghosh, S. S., & De Martino, F. (2019). Mapping the human subcortical auditory system using histology, postmortem MRI and in vivo MRI at 7T. ELife, 8. CLOCKSS.  doi: <a href="https://doi.org/10.7554/elife.48932">10.7554/eLife.48932</a>.</li>
<li>Fitzpatrick, P., Mitchell, T., Schmidt, R. C., Kennedy, D., & Frazier, J. A. (2019). Alpha band signatures of social synchrony. Neuroscience Letters, 699, 24‚Äì30.  doi: <a href="https://doi.org/10.1016/j.neulet.2019.01.037">10.1016/j.neulet.2019.01.037</a>.</li>
<li>Fenner, M., Crosas, M., Grethe, J. S., Kennedy, D., Hermjakob, H., Rocca-Serra, P., Durand, G., Berjon, R., Karcher, S., Martone, M., & Clark, T. (2019). A data citation roadmap for scholarly data repositories. Scientific Data, 6(1).  doi: <a href="https://doi.org/10.1038/s41597-019-0031-8">10.1038/s41597-019-0031-8</a>.</li>
<li>Millman, K. J., Brett, M., Barnowski, R., & Poline, J.-B. (2018). Teaching Computational Reproducibility for Neuroimaging. Frontiers in Neuroscience, 12.  doi: <a href="https://doi.org/10.3389/fnins.2018.00727">10.3389/fnins.2018.00727</a>.</li>
<li>Kim, Y.-M., Poline, J.-B., & Dumas, G. (2018). Experimenting with reproducibility: a case study of robustness in bioinformatics. GigaScience, 7(7).  doi: <a href="https://doi.org/10.1093/gigascience/giy077">10.1093/gigascience/giy077</a>.</li>
<li>Ozyurt, I. B., & Grethe, J. S. (2018). Foundry: a message-oriented, horizontally scalable ETL system for scientific data integration and enhancement. Database, 2018.  doi: <a href="https://doi.org/10.1093/database/bay130">10.1093/database/bay130</a>.</li>
<li>Guell, X., Schmahmann, J. D., Gabrieli, J. D., & Ghosh, S. S. (2018). Functional gradients of the cerebellum. ELife, 7. CLOCKSS.  doi: <a href="https://doi.org/10.7554/elife.36652">10.7554/eLife.36652</a>.</li>
<li>Kennedy, D. N. (2018). Neuroimaging Neuroinformatics: Sample Size and Other Evolutionary Topics. Neuroinformatics, 16(2), 149‚Äì150.  doi: <a href="https://doi.org/10.1007/s12021-018-9379-8">10.1007/s12021-018-9379-8</a>.</li>
<li>Wimalaratne, S. M., Juty, N., Kunze, J., Jan√©e, G., McMurry, J. A., Beard, N., Jimenez, R., Grethe, J. S., Hermjakob, H., Martone, M. E., & Clark, T. (2018). Uniform resolution of compact identifiers for biomedical data. Scientific Data, 5(1).  doi: <a href="https://doi.org/10.1038/sdata.2018.29">10.1038/sdata.2018.29</a>.</li>
<li>Solo, V., Poline, J.-B., Lindquist, M. A., Simpson, S. L., Bowman, F. D., Chung, M. K., & Cassidy, B. (2018). Connectivity in fMRI: Blind Spots and Breakthroughs. IEEE Transactions on Medical Imaging, 37(7), 1537‚Äì1550.  doi: <a href="https://doi.org/10.1109/tmi.2018.2831261">10.1109/TMI.2018.2831261</a>.</li>
<li>Kennedy, D. N. (2017). The Information Sharing Statement Grows Some Teeth. Neuroinformatics, 15(2), 113‚Äì114.  doi: <a href="https://doi.org/10.1007/s12021-017-9331-3">10.1007/s12021-017-9331-3</a>.</li>
<li>Nichols, T. E., Das, S., Eickhoff, S. B., Evans, A. C., Glatard, T., Hanke, M., Kriegeskorte, N., Milham, M. P., Poldrack, R. A., Poline, J.-B., Proal, E., Thirion, B., Van Essen, D. C., White, T., & Yeo, B. T. T. (2017). Best practices in data analysis and sharing in neuroimaging using MRI. Nature Neuroscience, 20(3), 299‚Äì303.  doi: <a href="https://doi.org/10.1038/nn.4500">10.1038/nn.4500</a>.</li>
<li>Poldrack, R. A., Baker, C. I., Durnez, J., Gorgolewski, K. J., Matthews, P. M., Munaf√≤, M. R., Nichols, T. E., Poline, J.-B., Vul, E., & Yarkoni, T. (2017). Scanning the horizon: towards transparent and reproducible neuroimaging research. Nature Reviews Neuroscience, 18(2), 115‚Äì126.  doi: <a href="https://doi.org/10.1038/nrn.2016.167">10.1038/nrn.2016.167</a>.</li>
<li>James, E. G., Leveille, S. G., Hausdorff, J. M., Travison, T., Kennedy, D. N., Tucker, K. L., Al Snih, S., Markides, K. S., & Bean, J. F. (2016). Rhythmic Interlimb Coordination Impairments and the Risk for Developing Mobility Limitations. The Journals of Gerontology Series A: Biological Sciences and Medical Sciences, glw236.  doi: <a href="https://doi.org/10.1093/gerona/glw236">10.1093/gerona/glw236</a>.</li>
<li>Ghosh, S. S., Poline, J.-B., Keator, D. B., Halchenko, Y. O., Thomas, A. G., Kessler, D. A., & Kennedy, D. N. (2017). A very simple, re-executable neuroimaging publication. F1000Research, 6, 124.  doi: <a href="https://doi.org/10.12688/f1000research.10783.2">10.12688/f1000research.10783.2</a>.</li>
<li>Sansone, S.-A., Gonzalez-Beltran, A., Rocca-Serra, P., Alter, G., Grethe, J. S., Xu, H., Fore, I. M., Lyle, J., Gururaj, A. E., Chen, X., Kim, H., Zong, N., Li, Y., Liu, R., Ozyurt, I. B., & Ohno-Machado, L. (2017). DATS, the data tag suite to enable discoverability of datasets. Scientific Data, 4(1).  doi: <a href="https://doi.org/10.1038/sdata.2017.59">10.1038/sdata.2017.59</a>.</li>
<li>Eglen, S. J., Marwick, B., Halchenko, Y. O., Hanke, M., Sufi, S., Gleeson, P., Silver, R. A., Davison, A. P., Lanyon, L., Abrams, M., Wachtler, T., Willshaw, D. J., Pouzat, C., & Poline, J.-B. (2017). Toward standard practices for sharing computer code and programs in neuroscience. Nature Neuroscience, 20(6), 770‚Äì773.  doi: <a href="https://doi.org/10.1038/nn.4550">10.1038/nn.4550</a>.</li>
<li>Huntenburg, J. M., Bazin, P.-L., Goulas, A., Tardif, C. L., Villringer, A., & Margulies, D. S. (2017). A Systematic Relationship Between Functional Connectivity and Intracortical Myelin in the Human Cerebral Cortex. Cerebral Cortex, 27(2), 981‚Äì997.  doi: <a href="https://doi.org/10.1093/cercor/bhx030">10.1093/cercor/bhx030</a>.</li>
<li>Irimia, A., Wei, S., Lu, N., Moore, C. M., & Kennedy, D. N. (2017). Mobile Monitoring of Traumatic Brain Injury in Older Adults: Challenges and Opportunities. Neuroinformatics, 15(3), 227‚Äì230.  doi: <a href="https://doi.org/10.1007/s12021-017-9335-z">10.1007/s12021-017-9335-z</a>.</li>
<li>Klein, A., Ghosh, S. S., Bao, F. S., Giard, J., H√§me, Y., Stavsky, E., Lee, N., Rossa, B., Reuter, M., Chaibub Neto, E., & Keshavan, A. (2017). Mindboggling morphometry of human brains. PLOS Computational Biology, 13(2), e1005350.  doi: <a href="https://doi.org/10.1371/journal.pcbi.1005350">10.1371/journal.pcbi.1005350</a>.</li>
<li>Gorgolewski, K. J., Alfaro-Almagro, F., Auer, T., Bellec, P., CapotƒÉ, M., Chakravarty, M. M., Churchill, N. W., Cohen, A. L., Craddock, R. C., Devenyi, G. A., Eklund, A., Esteban, O., Flandin, G., Ghosh, S. S., Guntupalli, J. S., Jenkinson, M., Keshavan, A., Kiar, G., Liem, F., ‚Ä¶ Poldrack, R. A. (2017). BIDS apps: Improving ease of use, accessibility, and reproducibility of neuroimaging data analysis methods. PLOS Computational Biology, 13(3), e1005209.  doi: <a href="https://doi.org/10.1371/journal.pcbi.1005209">10.1371/journal.pcbi.1005209</a>.</li>
<li>Maumet, C., Auer, T., Bowring, A., Chen, G., Das, S., Flandin, G., Ghosh, S., Glatard, T., Gorgolewski, K. J., Helmer, K. G., Jenkinson, M., Keator, D. B., Nichols, B. N., Poline, J.-B., Reynolds, R., Sochat, V., Turner, J., & Nichols, T. E. (2016). Sharing brain mapping statistical results with the neuroimaging data model. Scientific Data, 3(1).  doi: <a href="https://doi.org/10.1038/sdata.2016.102">10.1038/sdata.2016.102</a>.</li>
<li>Margulies, D. S., Ghosh, S. S., Goulas, A., Falkiewicz, M., Huntenburg, J. M., Langs, G., Bezgin, G., Eickhoff, S. B., Castellanos, F. X., Petrides, M., Jefferies, E., & Smallwood, J. (2016). Situating the default-mode network along a principal gradient of macroscale cortical organization. Proceedings of the National Academy of Sciences, 113(44), 12574‚Äì12579.  doi: <a href="https://doi.org/10.1073/pnas.1608282113">10.1073/pnas.1608282113</a>.</li>
<li>Gorgolewski, K. J., Auer, T., Calhoun, V. D., Craddock, R. C., Das, S., Duff, E. P., Flandin, G., Ghosh, S. S., Glatard, T., Halchenko, Y. O., Handwerker, D. A., Hanke, M., Keator, D., Li, X., Michael, Z., Maumet, C., Nichols, B. N., Nichols, T. E., Pellman, J., ‚Ä¶ Poldrack, R. A. (2016). The brain imaging data structure, a format for organizing and describing outputs of neuroimaging experiments. Scientific Data, 3(1).  doi: <a href="https://doi.org/10.1038/sdata.2016.44">10.1038/sdata.2016.44</a>.</li>
<li>Bandrowski, A. E., & Martone, M. E. (2016). RRIDs: A Simple Step toward Improving Reproducibility through Rigor and Transparency of Experimental Methods. Neuron, 90(3), 434‚Äì436.  doi: <a href="https://doi.org/10.1016/j.neuron.2016.04.030">10.1016/j.neuron.2016.04.030</a>.</li>
<li>Honor, L. B., Haselgrove, C., Frazier, J. A., & Kennedy, D. N. (2016). Data Citation in Neuroimaging: Proposed Best Practices for Data Identification and Attribution. Frontiers in Neuroinformatics, 10.  doi: <a href="https://doi.org/10.3389/fninf.2016.00034">10.3389/fninf.2016.00034</a>.</li>
</ul>

            	</div>
            </section>

            <footer class="site-footer">
                <div>
                    <ul class="list-inline quicklinks">
                    <li>Copyright &copy; ReproNim 2016-2024</div>
                    <li>Support: <a href="https://www.nibib.nih.gov/">NIH-NIBIB</a> <a href="https://www.nibib.nih.gov/research-funding/national-centers-biomedical-imaging-and-bioengineering">P41</a> <a href="https://projectreporter.nih.gov/project_info_description.cfm?aid=8999833">EB019936</a></li>
                    <li>Site theme: themewagon.com</li>
                    </ul>
                </div>
            </footer>
        </div>  <!-- wrapper -->

        <!-- js -->
        <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/js/bootstrap.min.js"></script>
    </body>
</html>
